account_comment_template/tests/test_account_move_report.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2017 Simone Rubino - Agile Business Group
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# Copyright 2021-2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests import Form, tagged
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged("post_install", "-at_install")
+class TestAccountInvoiceReport(AccountTestInvoicingCommon):
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+        cls.env = cls.env(
+            context=dict(
+                cls.env.context,
+                mail_create_nolog=True,
+                mail_create_nosubscribe=True,
+                mail_notrack=True,
+                no_reset_password=True,
+                tracking_disable=True,
+            )
+        )
+        cls.company = cls.env.ref("base.main_company")
+        cls.base_comment_model = cls.env["base.comment.template"]
+        cls.move_obj = cls.env["account.move"]
```

Comment:
You need get id of model in odoo. 

```suggestion
        cls.res_model_id = cls.env.ref("account.model_account_move")
```